### ISSUE TITLE
Weird string_mask behaviour on Mac OS X 10.9.5

### ISSUE THREAD

**Issue by mthssdrbrg** [rcrowley/certified#12]
My co-worker generated a CA and some certificates using the `openssl` version that comes bundled with Mac OS X 10.9.4, and everything worked fine. Me on the other hand, running Mac OS X 10.9.5, am unable to generate any certificates (using the same CA files and whatnot), and the failure message I receive is:

```
The localityName field needed to be the same in the
CA certificate (Göteborg) and the request (GÃ¶teborg).
```

After scavenging through some websites and man pages I found the option `string_mask` and tried to explicitly set the option in the configuration that's generated [here](https://github.com/rcrowley/certified/blob/master/bin/certified-csr#L101) to `default`, and after that it works fine (and the city name is displayed correctly).

Unfortunately I don't know what version of `openssl` that's used on Mac OS X 10.9.4, but on 10.9.5 it's `OpenSSL 0.9.8za`, though I believe Apple has rolled their own version of `openssl`.

Bottom line is whether it's possible to support different `string_mask` configuration values in `certified` in some non-invasive way? Or is there some other way we can work around this? (Besides forking the project and simply adding the configuration option...).

**Comment by nbrownus** [comment]
Just hit this, can confirm that `string_mask = utf8only` in `certified-csr` fixes this issue.

**Comment by mthssdrbrg** [comment]
Totally forgot about this issue as I haven't been generating certificates in quite a while (though we still have the `certified` setup).

Adding `string_mask = default` to the `[req]` configuration breaks the test suite for me as well. Using `utf8only` passes the test suite but fails to generate certificates for the CA setup that we currently have (using `default` instead works though).

**Comment by nbrownus** [comment]
I generated my pki on OSX and then moved to Linux and attempted to generate a cert, that's when I hit this bug. `string_mask = utf8only` worked for me on Linux, @mthssdrbrg what is your setup?
